mlflow/store/entities/paged_list.py

```diff
@@ -1,4 +1,9 @@
-class PagedList(list):
-    def __init__(self, items, token):
+from typing import TypeVar, List
+
+T = TypeVar("T")
+
+
+class PagedList(List[T]):
+    def __init__(self, items: List[T], token):
```

Comment:
Could we add a short docstring to describe these fields? e.g. something like

"""
Wrapper class around the base Python `List` type. Contains an additional `token`  string attribute
that can be passed to the pagination API that returned this list to fetch additional elements, if any
are available
"""

Reply:
Done!